upath/core.py

```diff
@@ -155,9 +128,10 @@ def __new__(cls, *args, **kwargs):
                 args = tuple(args_list)
                 self = cls._from_parts_init(args, init=False)
```

Comment:
Since you're at it, I'd rather inline the `_from_parts_init` method here, it's only used exactly once and explains itself much better than its name: `super()._from_parts(args, init=False)`

Reply:
That was already done in #43